test_psa_target.py

```diff
@@ -423,7 +438,9 @@ def _main():
     if build:
         test_spec = _init_test_spec(args)
         _build_regression_test(args, test_spec)
-        _build_compliance_test(args, test_spec)
+        # M2354 hasn't supported PSA compliance test yet.
```

Comment:
Please add this info to README.md:
* Section "Building the PSA Compliance Test suites" -> "Notes"
* Section "Expected test results": add a second bullet point

Reply:
Added